lib/templates/src/App.vue

```diff
@@ -11,18 +11,70 @@
       </Screen>
     </template>
 
-    <template #[screen]="{ trial }" v-for="screen in 2">
-      <AudioDiscriminationWithPriming
-        :priming-audio="trial.audio.primingAudio"
-        :trial-audio="trial.audio.trialAudio"
-        :option1="trial.options.option1"
-        :option2="trial.options.option2"
-        :key="screen"
-      >
-      </AudioDiscriminationWithPriming>
+    <template #1="{ nextScreen, addResult, trial }">
+      <Screen>
+        <ForcedChoiceInput
+          question="How did you like this?"
+          :options="[trial.options.option1, trial.options.option2]"
+          :answer.sync="answer"
+          @change:answer="
+            addResult({ answer, options: trial.options });
+            nextScreen();
+          "
+        />
+      </Screen>
+    </template>
+
+    <template #2="{ nextScreen, addResult, trial }">
+      <Screen>
+        <ImageSelectionInput
+          question="How did you like this?"
+          :options="[
+            { label: trial.options.option1, src: 'img/1.jpg' },
+            { label: trial.options.option2, src: 'img/2.jpg' }
+          ]"
```

Comment:
Wherever it made sense I tried to use arrays instead of direct props for options, so it's possible to add more than two or three options.